docs/content/operations/api-reference.md

```diff
@@ -230,6 +235,19 @@ Enables all segments of datasource which are not overshadowed by others.
 
 Enables a segment of a datasource.
 
+* `/druid/coordinator/v1/datasources/{dataSourceName}/markUnused`
+
+Marks segments unused for a datasource by interval or set of segment Ids. The request payload contains the interval or set of segment Ids to be marked unused.
+Either interval or segment ids should be provided, if both or none are provided in the payload , the API would throw an error (400 BAD REQUEST).Interval specifies the start and end times as IS0 8601 strings. `interval=(start/end)` where start and end both are inclusive.
+
```

Comment:
For the interval case, please mention that segments with intervals not wholly contained in the specified interval will not be affected.

Reply:
ah missed here, had added to proposal, updated now.